src/main/java/uk/gov/hmcts/reform/prl/models/dto/citizen/CitizenDocuments.java

```diff
@@ -30,14 +30,16 @@ public class CitizenDocuments {
 
     // Attributes for SOA
     private List<Document> applicantSoaPack; // if personal service - CA/CB -> Either applicant pack //
-    //private List<Document> unservedRespondentPack; // if personal service - CA/CB -> Either applicant pack //
     private List<Document> respondentSoaPack;
     private String servedParty;
+    private String whoIsResponsible;
     private boolean wasCafcassServed;
+    private boolean wasCafcassCymruServed;
```

Comment:
if these new attributes are related to notifications, we should move to citizennotifications class

Reply:
Hi @yogendraU, We still need these attributes here, these are for a particular order/soa.